rholang/src/main/scala/coop/rchain/rholang/interpreter/interpreter.scala

```diff
@@ -0,0 +1,11 @@
+package coop.rchain.rholang
+
+import scala.collection.immutable.HashMap
+
+package object interpreter {
+
+  type Data = Either[Quote, Par]
+
+  type Env[A] = HashMap[Int, A]
```

Comment:
If this is related to the Env above, you've made this infinitely harder to understand by putting them in separate files. Find a single home for them, and add comments about what you're doing.

Reply:
Definitions in package objects are package visible (you can't import a type alias). What if I rename Env to EnvOps and keep using Env[Par] in the body?

Reply:
This is closed. Thanks for working with me to make it more clear.